content/en/blog/_posts/2021-03-31-kube-state-metrics-goes-v-2-0.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: blog
+title: "kube-state-metrics goes v2.0"
+date: 2021-04-06
+slug: kube-state-metrics-v2-0
+---
+
+**Authors:** Lili Cosic (Red Hat), Frederic Branczyk (Polar Signals), Manuel Rüger
+
+## What?
+
+[Kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), a project under the Kubernetes organization, that generates Prometheus format metrics based on the current state of the Kubernetes native resources. It does this by listening to the Kubernetes API server and gathering information about resources and objects, e.g. Deployments, Pods, Services, and StatefulSets. A full list is available in the [documentation]( https://github.com/kubernetes/kube-state-metrics/tree/master/docs) of kube-state-metrics.
```

Comment:
```suggestion
[Kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), a project under the Kubernetes organization, generates Prometheus format metrics based on the current state of the Kubernetes native resources. It does this by listening to the Kubernetes API and gathering information about resources and objects, e.g. Deployments, Pods, Services, and StatefulSets. A full list is available in the [documentation]( https://github.com/kubernetes/kube-state-metrics/tree/master/docs) of kube-state-metrics.
```

Reply:
Also fixing incorrect whitespace, opening with an English word, and linking to the list of supported resources for metrics:
```suggestion
The [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) component of Kubernetes helps you to generate Prometheus format metrics based on the current state of your Kubernetes cluster's native resources. It does this by listening to the Kubernetes API and gathering information about resources and objects, e.g. Deployments, Pods, Services, and StatefulSets. A [full list](https://github.com/kubernetes/kube-state-metrics/tree/master/docs#exposed-metrics) is available in the kube-state-metrics [documentation](https://github.com/kubernetes/kube-state-metrics/tree/master/docs#readme).
```